### PR TITLE
Add tests for GitHub class

### LINKED ISSUE #70: Add tests for add command feature
Add more tests to comprehensively test add command feature

### PR DESCRIPTION
**Pull request by Incogdino**
Comprehensive of GitHub class to ensure that the methods perform as expected, correct errors are thrown.

In particular, the username regex needs to be changed to adhere to strict naming conventions of username by GitHub.

Closes #70

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Github.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public Github(String githubUsername) {
      * Returns if a given string is a valid email.
```
Comment (SwaminathanViswa) [inline src/main/java/seedu/address/model/person/Github.java:28]: You might want to change the java docs for this method. It still says `valid email` and not `valid Github Username`

**Review by SwaminathanViswa — COMMENTED** [review]
Once the typo has been fixed, its ready to be merged!

File: src/main/java/seedu/address/model/person/Github.java (hunk 1 above)
Comment (Incogdino) [reply src/main/java/seedu/address/model/person/Github.java:28]: Good point! I have changed it

**Review by SwaminathanViswa — APPROVED** [review]
LGTM!

**Pull request merged**